product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/01_vpc_endpoint.mdx

```diff
@@ -43,24 +43,29 @@ In the AWS account connected to BigAnimal, create an endpoint service to provide
 
 1. From the navigation pane on the left, select **Endpoint Services** under **Virtual Private Cloud**, and then select **Create endpoint service**.
 
-1. Enter a suitable name for the endpoint service.
+1. On the Create endpoint service page:
 
-1. Select **Network** for the load balancer type.
+   1. Enter a suitable name for the endpoint service.
 
-1. Under **Available load balancers**, select the network load balancer of the cluster to which you want to connect. 
- 
-1. Leave all the other fields with their default values and select **Create**. 
+   1. Select **Network** for the load balancer type.
+
+   1. Under **Available load balancers**, select the network load balancer of the cluster to which you want to connect. 
+
+   1. Leave all the other fields with their default values and select **Create**. 
 
 1. Under Details, note the **Service name** of the created endpoint service (for example, `com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc`). You will need the service name while creating a VPC endpoint.
+<img width="1285" alt="vpce-kwa-endpoint-copied" src="https://user-images.githubusercontent.com/90636096/184057438-3052939b-6b58-42dd-a923-3e1ce0f9ae92.png">
```

Comment:
I recommend taking out the next three screenshots. The first one doesn't show the service name in the text (`com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc`) and the Service name label is obscured by the Copied! popup. The second one should should Allow Principals highlighted in the Actions dropdown list and all three aren't really legible.